config/notifiers.go

```diff
@@ -351,3 +360,28 @@ func (c *PushoverConfig) UnmarshalYAML(unmarshal func(interface{}) error) error
 	}
 	return checkOverflow(c.XXX, "pushover config")
 }
+
+type AWSSNSConfig struct {
+	NotifierConfig `yaml:",inline"`
+
+	AWSRegion string `yaml:"aws_region"`
+	TopicArn  string `yaml:"topicarn"`
```

Comment:
The AWS SDK SNS Publish struct uses TopicArn. Leaving it TopicArn in the AWSSNSConfig struct keeps it consistent with the AWS SDK.